packages/client/wallets/aa/src/blockchain/wallets/FireblocksNCWallet.ts

```diff
@@ -34,7 +37,7 @@ export const FireblocksNCWallet = async (
         _deviceId = ncwData.deviceId;
         isNew = false;
     } else {
-        const ncwData = await crossmintService.getOrAssignWallet(userEmail);
+        const ncwData = localStorageRepository.ncwAddress ?? (await crossmintService.getOrAssignWallet(userEmail));
```

Comment:
this should be `localStorageRepository.ncwData ?? ...`

Reply:
Also a few lines down:
 `isNew = ncwData.isNew;`

I believe `isNew` is undefined, should we store it as well in localstorage?